packages/snaps-execution-environments/src/common/endowments/interval.ts

```diff
@@ -18,7 +20,10 @@ const createInterval = () => {
       );
     }
     const handle = Object.freeze({});
-    const platformHandle = setInterval(handler, timeout);
+    const platformHandle = setInterval(
+      handler,
+      Math.max(MINIMUM_INTERVAL, timeout ?? 0),
```

Comment:
Rather than using 10ms here if it's less, should we throw an error?

Reply:
We could.. But I'm not sure I'm a fan of changing the behaviour of the function like that. Could cause many weird issues with dependencies using `setInterval`?

Reply:
Fair point!